tubemq-server/src/main/java/org/apache/tubemq/server/broker/metadata/TopicMetadata.java

```diff
@@ -95,57 +97,63 @@ public TopicMetadata(final BrokerDefMetadata brokerDefMetadata, String topicMeta
             this.unflushInterval = Integer.parseInt(topicConfInfoArr[5]);
         }
         if (TStringUtils.isBlank(topicConfInfoArr[6])) {
-            this.deleteWhen = brokerDefMetadata.getDeleteWhen();
+            this.unflushSizeThreshold = brokerDefMetadata.getUnflushSizeThreshold();
```

Comment:
Easier for read, since it would be hard to realize the sequence if it evaluates topicConfInfoArr[7] and then a topicConfInfoArr[14] follows, people would feel weird about what's happening to 8-13 and guess whether they are no-in-use.